school-finder-frontend/app/index.html

```diff
@@ -1,10 +1,33 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>School Finder</title>
+    <title>School Finder</title>
+    <!-- Currently using CDN until we decide how we want to handle this -->
+    <script src='https://api.tiles.mapbox.com/mapbox.js/v2.1.5/mapbox.js'></script>
+    <link href='https://api.tiles.mapbox.com/mapbox.js/v2.1.5/mapbox.css' rel='stylesheet'/>
+    <script src="../node_modules/angular/angular.js"></script>
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+        }
+
+        #map {
+            position: absolute;
+            top: 0;
+            bottom: 0;
+            width: 100%;
+        }
+    </style>
 </head>
 <body>
-
-  <script src="../node_modules/angular/angular.js"></script>
+<div id='map'></div>
+<script>
+    // Currently using a personal access token, so feel free to update to an account created for Code for OKC
+    L.mapbox.accessToken = 'pk.eyJ1IjoibWJvY2t1cyIsImEiOiJGSVhxeUJzIn0.dFVS1mpYRCuR8fn_l0FWew';
```

Comment:
@joekarl We do have a Mapbox account for Code for OKC. Let me know how and where you want our public token.